AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/NoopSegmentationProcessor.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  NoopSegmentationProcessor.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// `NoopSegmentationProcessor` is a processor that does nothing except pass image frames in and out.
+/// This is used as a placeholder for implementations of `SegmentationProcessor` that cannot be initialized.
+public class NoopSegmentationProcessor : SegmentationProcessor {
+    private var buffer: UnsafeMutablePointer<UInt8> = UnsafeMutablePointer<UInt8>.allocate(capacity: 0)
+
+    public func initialize(height: Int, width: Int, channels: Int) -> Bool {
+        let capacity = height * width * Int(channels)
+        buffer = UnsafeMutablePointer.allocate(capacity: capacity)
+        return true
+    }
+
+    public func predict() -> Bool {
+        return true
+    }
+
+    public func getModelState() -> Int {
+        return 2 // LOADED
```

Comment:
Can we return `LOADED` instead?

Reply:
Changed it. As mentioned before, I preferred to keep the enum at project level but switched it to be exposed since it needs to be in bridging header.